include/inexor/vulkan-renderer/application.hpp

```diff
@@ -33,6 +35,9 @@ class Application : public VulkanRenderer {
 
     std::unique_ptr<input::KeyboardMouseInputData> m_input_data;
 
+    std::vector<std::shared_ptr<world::Cube>> m_worlds;
```

Comment:
A world should be a class in the long run, as it can contain the world bounding box and rotations.

Reply:
Maybe not in this pull request. This goes more towards a scene management pull request.

Reply:
Added feature request https://github.com/inexorgame/vulkan-renderer/issues/371